cmd/crowdsec-cli/keys.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/crowdsecurity/crowdsec/pkg/apiserver/middlewares"
+	"github.com/crowdsecurity/crowdsec/pkg/database"
+	"github.com/enescakir/emoji"
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var keyName string
+var keyIP string
+var keyLength int
+
+func NewKeysCmd() *cobra.Command {
+	/* ---- DECISIONS COMMAND */
+	var cmdKeys = &cobra.Command{
+		Use:   "keys [action]",
+		Short: "Manage local API keys",
+		Long: `
+API keys Management.
+
+To list/add/delete api keys
+`,
+		Example: `cscli keys [action]`,
+		Args:    cobra.MinimumNArgs(1),
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			var err error
+			dbClient, err = database.NewClient(csConfig.DbConfig)
+			if err != nil {
+				log.Fatalf("unable to create new database client: %s", err)
+			}
+		},
+	}
+
+	var cmdKeysList = &cobra.Command{
+		Use:     "list",
+		Short:   "List api keys",
+		Long:    `List `,
+		Example: `cscli keys list`,
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, arg []string) {
+			blockers, err := dbClient.ListBlockers()
+			if err != nil {
+				log.Errorf("unable to list blockers: %s", err)
+			}
+			if csConfig.Cscli.Output == "human" {
+
+				table := tablewriter.NewWriter(os.Stdout)
+				table.SetCenterSeparator("")
+				table.SetColumnSeparator("")
+
+				table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+				table.SetAlignment(tablewriter.ALIGN_LEFT)
+				table.SetHeader([]string{"Name", "IP Address", "Valid", "Last API pull"})
+				for _, b := range blockers {
+					var revoked string
+					if !b.Revoked {
+						revoked = fmt.Sprintf("%s", emoji.CheckMark)
+					} else {
+						revoked = fmt.Sprintf("%s", emoji.Prohibited)
+					}
+					table.Append([]string{b.Name, b.IPAddress, revoked, fmt.Sprintf("%s", b.LastPull.Format(time.RFC3339))})
+				}
+				table.Render()
+			} else if csConfig.Cscli.Output == "json" {
+				x, err := json.MarshalIndent(blockers, "", " ")
+				if err != nil {
+					log.Fatalf("failed to unmarshal")
+				}
+				fmt.Printf("%s", string(x))
+			} else if csConfig.Cscli.Output == "raw" {
+				for _, b := range blockers {
```

Comment:
beware, here you separate "raw" output with spaces, but it was with `,` in the `ban list` one. I think you want as well to display the "column" names.